activesupport/activesupport.gemspec

```diff
@@ -42,5 +42,5 @@ Gem::Specification.new do |s|
   s.add_dependency "base64"
   s.add_dependency "drb"
   s.add_dependency "bigdecimal"
-  s.add_dependency "logger"
+  s.add_dependency "logger",          ">= 1.4.2"
```

Comment:
I've seen formatting rules where the column width should match subsequent lines but given there is multiple lines between them here, is it still required?

Reply:
Yeah, it probably isn't necessary 👍